### PR TITLE
A shrestha 3

### PR DESCRIPTION
**Pull request by anujmani**
Add new line in READme

### CODE REVIEW

File: gradlew [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 #
-# Copyright � 2015-2021 the original authors.
+# Copyright © 2015-2021 the original authors.
```
Comment (devintern-esewa) [inline gradlew:4]: PLease revert the changes on this file

File: gradlew [hunk 2]
```diff
@@ -32,10 +32,10 @@
 #       Busybox and similar reduced shells will NOT work, because this script
 #       requires all of these POSIX shell features:
 #         * functions;
-#         * expansions �$var�, �${var}�, �${var:-default}�, �${var+SET}�,
```
Comment (RujanRanjitkar) [inline gradlew:35]: Please dont commit extra files

File: gradlew.bat [hunk 3]
```diff
@@ -40,7 +40,7 @@ if defined JAVA_HOME goto findJavaFromJavaHome
 
 set JAVA_EXE=java.exe
 %JAVA_EXE% -version >NUL 2>&1
-if "%ERRORLEVEL%" == "0" goto execute
```
Comment (GyalbuSherpa5) [inline gradlew.bat:43]: please revert these changes

**Review by GyalbuSherpa5 — APPROVED** [review]
ok

**Review by RujanRanjitkar — APPROVED** [review]

**Review by sumanpaudel333 — APPROVED** [review]
